organization.go

```diff
@@ -197,6 +199,9 @@ type OrganizationCreateOptions struct {
 
 	// Optional: AllowForceDeleteWorkspaces toggles behavior of allowing workspace admins to delete workspaces with resources under management.
 	AllowForceDeleteWorkspaces *bool `jsonapi:"attr,allow-force-delete-workspaces,omitempty"`
+
+	// Optional: DefaultExecutionMode the default execution mode for workspaces
+	DefaultExecutionMode *string `jsonapi:"attr,default-execution-mode,omitempty"`
```

Comment:
It looks like we are missing DefaultAgentPool in the create options

Reply:
Yeah, that's actually by design. You need an organization before you can get an agent pool.